Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -282,76 +284,19 @@ void EncodeToRamFromTexture(u8* dest_ptr, const EFBCopyParams& params, u32 nativ
   texconv_shader.program.Bind();
   glUniform4i(texconv_shader.copy_position_uniform, src_rect.left, src_rect.top, native_width,
               scale_by_half ? 2 : 1);
+  glUniform1f(texconv_shader.y_scale_uniform, params.y_scale);
 
   const GLuint read_texture = params.depth ?
                                   FramebufferManager::ResolveAndGetDepthTarget(src_rect) :
                                   FramebufferManager::ResolveAndGetRenderTarget(src_rect);
 
   EncodeToRamUsingShader(read_texture, dest_ptr, bytes_per_row, num_blocks_y, memory_stride,
-                         scale_by_half && !params.depth);
+                         scale_by_half && !params.depth, params.y_scale);
 
   FramebufferManager::SetFramebuffer(0);
   g_renderer->RestoreAPIState();
 }
 
-void EncodeToRamYUYV(GLuint srcTexture, const TargetRectangle& sourceRc, u8* destAddr, u32 dstWidth,
-                     u32 dstStride, u32 dstHeight)
-{
-  g_renderer->ResetAPIState();
-
-  s_rgbToYuyvProgram.Bind();
```

Comment:
Might you also drop those shaders? s_rgbToYuyvProgram and s_yuyvToRgbProgram?

Reply:
Yeah, I noticed this last night but didn't get around to removing it.  I'll remove those tonight.  Thanks!